packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -128,6 +137,17 @@ export default {
         closeButtonCopy: {
             type: String,
             default: 'Close modal'
+        },
+
+        title: {
+            type: String,
+            default: ''
+        },
+
+        titleHtmlTag: {
+            type: String,
+            default: 'h3',
+            validator: value => ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'].includes(value)
```

Comment:
I'd consider removing a couple of these  – I'm not sure we'd ever need a modal to have a heading of `h5`/`h6`?